### ISSUE TITLE
Billable total

### ISSUE THREAD

**Issue by wittwerch** [hundertzehn/mocoapp-api-docs#115]
It would be great to either
-  get the billable total (showed on /projects at the top) from our Moco app.
- get the billable items (and their total) per project (from /projects/1234/invoices)

We have build a small dashboard and would like to show that value compared to the costs set in Moco, to transparently show how we're doing :)

**Comment by agileapplications** [comment]
Hi @wittwerch, nice idea. Send a screenshot if possible (with fake numbers) :-) 

We bundle a few API updates and also consider your request. I hope to fit it in in February :+1: 

Best, Tobias

**Comment by agileapplications** [comment]
@wittwerch Can you send the screenshot I asked? That would help us understand your request. Thanks!

**Comment by wittwerch** [comment]
Hi @agileapplications 

Sorry for not coming back early on that. We do use the invoices from Moco to report our gross profit on a per month base. 

<img width="1000" alt="2021-02-16_07-41-59" src="https://user-images.githubusercontent.com/1575445/108027903-3335ca80-702b-11eb-8efe-7b5fd0e945fa.png">

During a month it would be nice to show the current progress aka the billable total. We usually look at that number, because it's projection of how much we're going to be able to bill (aka gross profit).

<img width="1000" alt="2021-02-16_07-48-39" src="https://user-images.githubusercontent.com/1575445/108028084-89a30900-702b-11eb-8f61-8b658fdafc8b.png">

So in the end it would be nice to have that number per project, so that we can sum it up and show "temporarily" in the current month. Basically till we make the invoices and get the hard facts :)

**Comment by nethad** [comment]
@wittwerch thanks for the clarification and the screenshots. This got delayed a little internally, but we will get back to you!

**Comment by agileapplications** [comment]
@wittwerch So from the MOCO perspective the green number total on top of the project list (like on your screenshot) and the green one next to each project, correct?

**Comment by wittwerch** [comment]
Yes, exactly!

**Comment by agileapplications** [comment]
We haven't yet hat the opportunity to work on this @wittwerch. Getting back to it...

**Comment by wittwerch** [comment]
Hi @agileapplications, I just wanted to ask if that is still on the roadmap?

**Comment by agileapplications** [comment]
@wittwerch We prepared a new `/report` endpoint to add also this one. Sorry for being unresponsive here. It's the first request yet and we have to provide a fast solution (performance-wise) to put this online. It will take more time unfortunately.

**Comment by wittwerch** [comment]
@agileapplications Thanks for the update, appreciate it! Looking forward to the new API :)

**Comment by acydburn** [comment]
It seems like the projects/{id}/report endpoint returns wrong numbers if several hours are marked as "bereits abgerechnet". Actually it would be nice if the values within the projects "Abrechnung" Tab would be reflected correctly:

Abrechenbare Positionen:
- x,xx Stunden
- xxx EUR

At the moment i think only billable_hours are returned anyways. The amount is not able to be calculated easily.

I think this is also what the original request is about... to simply return these two values for all projects. The current billable hours and amount (no date selection or anything).

**Comment by wittwerch** [comment]
> I think this is also what the original request is about... to simply return these two values for all projects. The current billable hours and amount (no date selection or anything).

Yes, exactly. We basically want a projection of the revenue we are going to do in that month.
@agileapplications Is that feature on the roadmap somehow? Would appreciate a short feedback on this.

**Comment by Schaukelking** [comment]
Hey there, apologies for the late response.

We're currently consolidating our REST API feedback channels and therefore we’re deprecating the Issue Board in this repository. For any other issues or feature requests, please contact us directly via the support.

Our Product Team has already received this request and will take it into consideration.

Thank you for your understanding!
